alacritty.yml

```diff
@@ -39,6 +39,9 @@ window:
   #     - none: Neither borders nor title bar
   decorations: full
 
+  # When true, alacritty starts maximized.
+  maximized: false
```

Comment:
Thinking about this again, I feel like `start_maximized` might be a better name for this config setting (the way it was in the original PR). It's just a little more telling about what's going on exactly.

Reply:
I thought it was similar to the `title` where its implicit that all these settings are just to start

Reply:
I was even considering renaming it to `maximize`, but I won't bikeshed too long if anyone else has strong feelings

Reply:
The `title` is a bit different, since it might actually be permanent if the dynamic title is disabled.

Maximized is nice and short, but with `start_maximized` there's no confusion if a call to `maximized()` is checking if it's currently maximized, or if that's just the default launch configuration.